docs/source/release_notes/lts/52_0.rst

```diff
@@ -163,3 +163,176 @@ When compared to the last LTS, we had:
 - 69 closed GitHub issues.
 - 34 contributors from at least 12 different companies, 26 of them
   contributing for the fist time to the project.
+
+
+Switching from 36.4 to 52.0
+---------------------------
+
+You already know what new features you might expect, but let's emphasize the
+main changes required to your workflows/tests when switching from 36.4 to
+52.0
+
+Installation
+^^^^^^^^^^^^
+
+All the previously supported ways to install Avocado are still valid and
+few new ones were added, but beware that Avocado was split into several
+optional plugins so you might want to adjust your scripts/workflows.
+
+* Multiplexer (the YAML parser which used to generate variants) was
+  turned into an optional plugin ``yaml_to_mux`` also known as
+  ``avocado_framework_plugin_varianter_yaml_to_mux``. Without it Avocado
+  does not require PyYAML, but you need it to support the parsing of
+  YAML files to variants (unless you use a different plugin with similar
+  functionality, which is now also possible).
+* The HTML result plugin is now also an optional plugin so one has to
+  install it separately.
+* The remote execution features (``--remote-hostname``, ``--vm-domain``,
+  ``--docker``) were also turned into optional plugins so if you need
+  those you need to install them separately.
+* Support for virtual environment (``venv``) was greatly improved and we
+  do encourage people who want to use ``pip`` to do that via this method.
+
+As for the supported ways:
+
+* Fedora/RHEL can use our custom repositories, either LTS-only or all
+  releases. Note that latest versions (non-lts) are also available
+  directly in Fedora and also in EPEL.
+* OpenSUSE - Ships the 36 LTS versions, hopefully they'll start
+  shipping the 52 ones as well (but we are not in charge of that
+  process)
+* Debian - The ``contrib/packages/debian`` script is still available,
+  although un-maintained for a long time
+* PyPI/pip - Avocado as well as all optional plugins are available in
+  PyPI and can be installed via ``pip install avocado-framework\*``,
+  or selectively one by one.
+* From source - Makefile target ``install`` is still available but it
```

Comment:
I don't consider this for public use, it's meant for developers... so I'd prefer keeping it this way, it's not "how to install" section, but what changed concerning installation.